src/core/constants.js

```diff
@@ -22,8 +22,18 @@ export const VERSION =
  */
 export const P2D = 'p2d';
 /**
- * One of the two render modes in p5.js: P2D (default renderer) and WEBGL
- * Enables 3D render by introducing the third dimension: Z
+ * One of the two render modes in p5.js, most known for 3D rendering.
```

Comment:
A small nit but can we phrase this a little differently to reflect WEBGL's usage rather than it's notoriety. 

`One of the two render modes in p5.js, used for computationally intensive tasks like 3d rendering and drawing shaders.`